packages/core/src/Button/Button.styles.js

```diff
@@ -1,23 +1,228 @@
 import css from 'styled-jsx/css'
```

Comment:
We have quite some duplicate selectors in this file. I would love it if we could deduplicate them, would really simplify reading through the styles.